src/oscar/apps/offer/abstract_models.py

```diff
@@ -111,7 +111,7 @@ class AbstractConditionalOffer(models.Model):
                     "a user for the duration of their session")),
     )
     offer_type = models.CharField(
-        _("Type"), choices=TYPE_CHOICES, default=SITE, max_length=128)
```

Comment:
I feel like it may be better to leave this default intact, as projects may be relying on it in ways we haven't anticipated, and it doesn't seem like a necessary change to support the feature we're adding. Is there a reason it is necessary?